Jenkinsfile_CNP

```diff
@@ -9,7 +9,13 @@ List<LinkedHashMap<String, Object>> secrets = [
   secret('AatTestUserUsername', 'SMOKE_TEST_CITIZEN_USERNAME'),
   secret('AatTestUserUsername', 'SMOKE_TEST_SOLICITOR_USERNAME'),
   secret('AatTestUserPassword', 'SMOKE_TEST_USER_PASSWORD'),
-  secret('AatTestUserEmailPattern', 'GENERATED_USER_EMAIL_PATTERN')
+  secret('AatTestUserEmailPattern', 'GENERATED_USER_EMAIL_PATTERN'),
+  secret('ClaimStoreDbPassword', 'CLAIM_STORE_DB_PASSWORD'),
+  secret('ClaimStoreDbHost', 'CLAIM_STORE_DB_HOST'),
```

Comment:
We do not have to retrieve HOST name from vault.

Reply:
it was easier way as host for different env are not the same:
prod-data-lb.moneyclaim.reform.hmcts.net
test-data-lb.moneyclaim.reform.hmcts.net

Reply:
unless we only want to execute migration on `after('deploy:prod')` and we don't care for aat? [just to make sure it works well before production data]